bin/near

```diff
@@ -83,6 +83,23 @@ const createAccount = {
   handler: (argv) => exitOnError(main.createAccount(argv))
 };
 
+const login = {
+  command: 'login <accountId>',
+  desc: 'create a developer account',
+  builder: (yargs) => yargs
+    .option('walletUrl', {
+      desc: 'Wallet URL to use for login',
+      type: 'string',
+      default: 'https://wallet.nearprotocol.com',
+    })
+    .option('accountId', {
```

Comment:
shouldn't it be chosen from wallet UI?

Reply:
We need to know accountId on the shell side, so it's either passing accountId to wallet and then failing if it can't be used, or setting up some callback mechanism to shell. 1st option seems much simpler

Reply:
1st option isn't good UX for when you don't have any account yet.

It makes sense though as a shorthand to pre-select existing account in wallet.